src/Message/KPRefundOrderCheckResponse.php

```diff
@@ -31,6 +31,11 @@ public function getData(): array
     public function isSuccessful()
     {
         $klarnaOrder = $this->responseBody;
+
+        if ($this->response->getStatusCode() != 200) {
+            return $this->response->getStatusCode();
+        }
+
```

Comment:
Looks like this function is meant to return a bool, this is now returning a bool or integer?

Reply:
Good spot, this can just be `return false`